_partials/git.md

```diff
@@ -0,0 +1,47 @@
+## Git
+
+### Installation
+
+[`git`](https://git-scm.com/) is a command line software used for version control.
+
+To install `git`:
+- Open a terminal
+- Copy and paste the following commands:
+
+```bash
+sudo apt update
+sudo apt install -y git
+````
+
+These commands will ask for your password: type it in.
+
+:warning: When you type your password, nothing will show up on the screen, **that's normal**. This is a security feature to mask not only your password as a whole but also its length. Just type in your password and when you're done, press `ENTER`.
+
+### GitHub CLI
+
+Let's now install [GitHub official CLI](https://cli.github.com) (Command Line Interface). It's a software used to interact with your GitHub account via the command line.
+
+In you terminal, copy-paste the following commands and type in your password if asked:
+
+```bash
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C99B11DEB97541F0
```

Comment:
❓ Should we keep the `ubuntu_` prefix in the partial name because of the `apt` commands?

Reply:
I don't think so. Here is the convention I've followed: 
- Partial prefixed with the OS if it's specific to only one OS
- No prefix otherwise

Here, the git partial is common to Windows and Ubuntu so setup so no prefix.

Reply:
Thank you for clarifying 🙏 
I tried to take this into account in the [contributing guidelines](https://github.com/lewagon/setup/commit/9220882b7425c7e2780cc40e9c29674078783632), let me know what you think!

Reply:
Nice. I've just added an extra sentence on the chinese translated versions and fixed a typo :heavy_check_mark: